model/vfs/vfsafero/fsck.go

```diff
@@ -41,12 +47,16 @@ func (afs *aferoVFS) CheckFilesConsistency(accumulate func(log *vfs.FsckLog)) er
 	if err != nil {
 		return err
 	}
-	return afs.checkFiles(entries, accumulate)
+	return afs.checkFiles(entries, accumulate, failFast)
 }
 
-func (afs *aferoVFS) checkFiles(entries map[string]*vfs.TreeFile, accumulate func(log *vfs.FsckLog)) (err error) {
+func (afs *aferoVFS) checkFiles(
+	entries map[string]*vfs.TreeFile,
+	accumulate func(log *vfs.FsckLog),
+	failFast bool,
+) error {
```

Comment:
Not sure if it was intended or not (my go skills are so bad) but `(err error)` has been replaced with `error`

Reply:
It is intended: I try to avoid named returns as the experience has shown that they make the code harder to review and change.